products/eslint.md

```diff
@@ -0,0 +1,103 @@
+---
+title: eslint
+category: framework
+iconSlug: eslint
+permalink: /eslint
+releasePolicyLink: https://eslint.org/version-support/
+changelogTemplate: "https://github.com/eslint/eslint/releases/tag/v__LATEST__"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn:  Maintenance Support
+eoasColumn: Active Support
+tags: herodevs
+customColumns:
+  - property: eoesProvider
+    position: before-latest-column
+    label: Extended Support Provider
+    description: Companies that provide extended EOL support for eslint.
+
+auto:
+  methods:
+    - npm: eslint
+
+identifiers:
+  - repology: eslint
+
+releases:
+  - releaseCycle: "9"
+    releaseDate: 2024-04-05
+    eol: false
+    eoas: false
+    latest: "9.9.0"
+    latestReleaseDate: 2024-08-08
+
+  - releaseCycle: "8"
+    releaseDate: 2021-10-09
+    eol: 2024-10-05
+    eoas: 2024-04-05
+    eoesProvider: HeroDevs
+    latest: "8.57.0"
+    latestReleaseDate: 2024-02-23
+
+  - releaseCycle: "7"
+    releaseDate: 2020-05-08
+    eol: 2022-04-09
+    eoas: 2021-10-09
+    eoesProvider: HeroDevs
+    latest: "7.32.0"
+    latestReleaseDate: 2021-07-30
+
+  - releaseCycle: "6"
+    releaseDate: 2019-06-21
+    eol: 2020-11-08
+    eoas: 2020-05-08
+    eoesProvider: HeroDevs
+    latest: "6.8.0"
+    latestReleaseDate: 2019-12-20
+
+  - releaseCycle: "5"
+    releaseDate: 2018-06-22
+    eol: 2019-12-21
+    eoas: 2019-06-21
+    eoesProvider: HeroDevs
+    latest: "5.16.0"
+    latestReleaseDate: 2019-03-29
+
+  - releaseCycle: "4"
+    releaseDate: 2017-06-11
+    eol: 2019-12-22
+    eoas: 2018-06-22
+    eoesProvider: HeroDevs
```

Comment:
```suggestion
```

According to https://www.herodevs.com/ extended support is provided only for the latest 8.x version.